README.md

```diff
@@ -1,7 +1,7 @@
 # SORSE Website repository
 
-This repository contains the source code for the Series of Online Research Software
-Events 2020. It is based on the [Minimal Mistakes Jekyll theme](https://github.com/mmistakes/minimal-mistakes)
+This repository contains the source code for the [Series of Online Research Software
+Events (SORSE) 2020](https://sorse.github.io). It is based on the [Minimal Mistakes Jekyll theme](https://github.com/mmistakes/minimal-mistakes)
```

Comment:
```suggestion
Events (SORSE)](https://sorse.github.io). It is based on the [Minimal Mistakes Jekyll theme](https://github.com/mmistakes/minimal-mistakes)
```

Thanks @danielskatz! We chose to not use 2020 at the moment, as this series probably extends into the year 2021.

Reply:
Yes, I was wondering about that.